frontend/src/scenes/sessions/SessionsTable.tsx

```diff
@@ -98,17 +95,23 @@ export function SessionsTable({ personIds, isPersonPage = false }: SessionsTable
             },
             ellipsis: true,
         },
-    ]
-
-    if (!user?.is_multi_tenancy || featureFlags['session-recording-player']) {
-        columns.push({
-            title: 'Play Session',
+        {
+            title: (
+                <span>
+                    <Tooltip title="Enable session recording from Project > Settings">
```

Comment:
Maybe add a direct link to the settings page from here?

Reply:
Not sure what you have in mind design-wise - mind elaborating?

Reply:
Thinking just turn "Settings" into a link? or even if you click the tooltip body?